src/style/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  margin: 16px 16px 0;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    color: #e5e5e5;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center center;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
the key idea is to be the initial functionality of `stars` BEM-block (i.e. with all those modifiers), and then apply for-loop, see the checklist, p.12